.github/workflows/_themes-integration.yml

```diff
@@ -0,0 +1,72 @@
+name: Integration
```

Comment:
I would suggest to use _Themes integration_ as a name for this workflow - this way we will be consistent with the other workflows naming, where the actual workflow name matches the .yml file name (but without dashes and underscores)